### PR TITLE
Create logging functionality

### PR DESCRIPTION
**Pull request by Mlastawi**
Added configuration for logging and directory to store logs.
Function `get_logger` gets logger from configuration file `config/logging.conf`. If logger name is not passed to function `acid` logger is used by default.
Logs are stored in `logs/acid.log` and `logs/acid.log.debug` files.

Logger has hierarchical structure, so (by default) all logs passed to `acid.feature.name` logger are passed to `acid.feature` and `acid` loggers as well. Users can add their own loggers and handlers in addition to basic `acid` logger.

### CODE REVIEW

File: logs/README [hunk 1]
```diff
@@ -0,0 +1 @@
+THE LOGS FROM ACID AND IT'S FEATURES WILL BE STORED HERE
```
Comment (xbend) [inline logs/README]: Need to remake this, and have the logging code in ACID be ready for a filepath specification that does not exist in the system.
look here: https://stackoverflow.com/questions/20666764/python-logging-how-to-ensure-logfile-directory-is-created?rq=1

File: acid/utils.py [hunk 2]
```diff
@@ -7,3 +8,10 @@ def pipe_intersect(pipelines_config, pipelines_json):
 
     return [pipeline['name'] for pipeline in pipelines_json
             if pipeline['name'] in pipelines_config]
+
+
+def get_feature_logger(feature_name=None):
```
Comment (MichalMilewicz) [inline acid/utils.py]: please change name for something more universal like: app_loger() etc.

File: config/logging.conf.sample [hunk 3]
```diff
@@ -0,0 +1,44 @@
+[loggers]
```
Comment (MichalMilewicz) [inline config/logging.conf.sample]: please add comments to each point in configuration. Comments helpful for someone who gonna create config

File: acid/utils.py (hunk 2 above)
Comment (xbend) [reply acid/utils.py]: `get_feature_logger` -> `get_logger`

**Pull request merged**